Package.swift

```diff
@@ -39,8 +39,7 @@ let package = Package(
         .package(url: "https://github.com/handya/OhhAuth.git", from: "1.4.0"),
         .package(url: "https://github.com/scinfu/SwiftSoup.git", from: "2.3.2"),
         .package(name: "SwiftPM",
-                 url: "https://github.com/apple/swift-package-manager.git",
-                 .revision("swift-DEVELOPMENT-SNAPSHOT-2021-05-04-a"))
+                 url: "https://github.com/finestructure/swift-package-manager.git", .branch("signer-default-observability-scope"))
```

Comment:
My branch contains a workaround for accessing `PackageCollectionSigning`. I've opened a PR against SPM here: https://github.com/apple/swift-package-manager/pull/3836